src/index.html

```diff
@@ -16,17 +16,28 @@
       href="style.css"
     />
   </head>
-  <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+  <body class="body">
+    <form
+      data-qa="big"
+      action="#"
+    >
+      <input
+        class="search_input_big form"
```

Comment:
The class name `search_input_big form` could be improved. Avoid using tag names in class names (like `form`). Also, avoid specifying styles in class names (like `big`). Instead, consider something like `search-input search-input--large`.